client/src/util/browser.ts

```diff
@@ -32,13 +32,14 @@ export default {
     return width <= 599;
   },
 
-  isIE() {
+  isIE(): boolean {
     if(typeof window != 'undefined') {
       return /MSIE \d|Trident.*rv:/.test(window.navigator.userAgent);
     }
+    return false;
   },
 
-  getParameterByName(name, url) {
+  getParameterByName(name: string, url: string): string|null {
```

Comment:
As far as I can tell, neither this function nor `isIE()` are called from anywhere in the code, so we might just want to get rid of them?

Note also that `getParameterByName` is also somewhat obviated by the existence of [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams); while it doesn't appear to exist on IE11, I suspect there's a polyfill available.